### PR TITLE
Update QUEST_JOBSTEP.tsv [Lines 1880 - 2079]

### PR DESCRIPTION
**Pull request by Vilifisaur**

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -1877,32 +1877,32 @@ QUEST_JOBSTEP_20150317_001876	Cleric Master told you that the herbs are running
 QUEST_JOBSTEP_20150317_001877	Return to Master Cleric
 QUEST_JOBSTEP_20150317_001878	Gathered wild flower staman. Give it to Cleric Master.
 QUEST_JOBSTEP_20150317_001879	Wild Flower Staman
-QUEST_JOBSTEP_20150317_001880	Retrieve the lost holy materials in town center
+QUEST_JOBSTEP_20150317_001880	Retrieve the lost Holy Materials in town center
 QUEST_JOBSTEP_20150317_001881	Krivis Master says the Holy Materials were stolen during a raid on its way to Fedimian. Find the lost Holy Materials in town center and bring it to Krivis Master.
 QUEST_JOBSTEP_20150317_001882	Deliver to Krivis Master
-QUEST_JOBSTEP_20150317_001883	Found the lost Holy materials. Give it to Krivis Master.
+QUEST_JOBSTEP_20150317_001883	Found the lost Holy Materials. Gave it to Krivis Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Gave? No, it's 'give'. It's not past tense.

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -1877,32 +1877,32 @@ QUEST_JOBSTEP_20150317_001876	Cleric Master told you that the herbs are running
...
 QUEST_JOBSTEP_20150317_001884	Find %s from town center
 QUEST_JOBSTEP_20150317_001885	Defeat Tutu
-QUEST_JOBSTEP_20150317_001886	Talk to Priest master in Klaipeda.
+QUEST_JOBSTEP_20150317_001886	Talk to Priest Master in Klaipeda.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Talk to **the** Priest Master.

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -1877,32 +1877,32 @@ QUEST_JOBSTEP_20150317_001876	Cleric Master told you that the herbs are running
...
-QUEST_JOBSTEP_20150317_001895	Defeat Stone Whale and get %s
+QUEST_JOBSTEP_20150317_001894	Got the essences of Petrified Whales. Bring it to Bokor Master.
+QUEST_JOBSTEP_20150317_001895	Defeat Petrified Whales and get %s
 QUEST_JOBSTEP_20150317_001896	Go meet the sculptor Tesla in Rhombuspaving Dale
 QUEST_JOBSTEP_20150317_001897	Get piece materials from town center
-QUEST_JOBSTEP_20150317_001898	Sculptor Tesla says he'll take you as a disciple if you get him the materials he want. Get large wood from the courtyard of town center.
+QUEST_JOBSTEP_20150317_001898	Sculptor Tesla says he'll take you as a disciple if you get him the materials he wants. Get large wood from the courtyard of town center.
 QUEST_JOBSTEP_20150317_001899	Give it to Sculptor Tesla
-QUEST_JOBSTEP_20150317_001900	Got the large woods. Give it to Tesla.
+QUEST_JOBSTEP_20150317_001900	Got large wood. Gave it to Sculptor Tesla.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Gave? No, it's 'give'. It's not past tense.

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -1928,51 +1928,51 @@ QUEST_JOBSTEP_20150317_001927	Defeat Denoptics that suddenly appeared
...
 QUEST_JOBSTEP_20150317_001942	Gather materials and make %s
 QUEST_JOBSTEP_20150317_001943	Go to Peltasta Master in Klaipeda
-QUEST_JOBSTEP_20150317_001944	Defeat the Vubbes in Crystal Mine 1st Gallery
-QUEST_JOBSTEP_20150317_001945	You need to pass the first assignment in order to become a Peltasta. Defeat the Vubbes invading Crystal Mines 1st Floor.
+QUEST_JOBSTEP_20150317_001944	Defeat the Vubbes on the 1st Floor of Crystal Mines
+QUEST_JOBSTEP_20150317_001945	You need to pass the first assignment in order to become a Peltasta. Defeat the Vubbes invading the 1st Floor of Crystal Mines.
 QUEST_JOBSTEP_20150317_001946	Defeated all the Vubbes. Please report back to the Peltasta master.
-QUEST_JOBSTEP_20150317_001947	Kill the Vubbes in Crytal Mines 1st Gallery
-QUEST_JOBSTEP_20150317_001948	You need to pass the first assignment in order to become a Highlander. Defeat the Vubbes invading Crystal Mines 1st Floor.
+QUEST_JOBSTEP_20150317_001947	Kill the Vubbes on the 1st Floor of Crytal Mines
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Crytal -> Crystal

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -1928,51 +1928,51 @@ QUEST_JOBSTEP_20150317_001927	Defeat Denoptics that suddenly appeared
...
 QUEST_JOBSTEP_20150317_001952	Help the research of Bokor Master
-QUEST_JOBSTEP_20150317_001953	Pyromancer Master says he'll accept you if you help with his research. Go to Bokor Master who is working on the research together and help her study.
+QUEST_JOBSTEP_20150317_001953	Pyromancer Master says he'll accept you if you help with his research. Go to Bokor Master who is working on the research together and help with her study.
 QUEST_JOBSTEP_20150317_001954	Helped Bokor Master's research. Report to Pyromancer Master.
 QUEST_JOBSTEP_20150317_001955	Summoned Evil
-QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer master in Klaipeda.
-QUEST_JOBSTEP_20150317_001957	Open the box of Pyromancer Mastert
-QUEST_JOBSTEP_20150317_001958	Cryomancer Master told you to open the box in Pyromancer Master's room as a condition to accept you. Something about it feels uncomfortable but open the box anyway.
-QUEST_JOBSTEP_20150317_001959	Report to Cryomancer master
+QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer Master in Klaipeda.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Talk to **the**

**Comment by Vilifisaur** [comment]
Ok thanks for the corrections. I saw how some of the lines didn't include "the" when talking with the Masters so I left them out.

**Comment by Sunaries** [comment]
Use the definite article (the) when you need to visit a Master. 
You edited it to Crystal Mines, but it is Crystal Mine AFAIK.

**Comment by Vilifisaur** [comment]
Is it suppose to be Crystal Mine? Because in other parts of the Quest_Jobstep it has Crystal Mines. Also should I leave the original text alone when it has ex: "in Crystal Mine 1st Floor" or change it to match other sections ex: "on the 1st Floor of Crystal Mine."

**Comment by imcgames** [comment]
If you plan to change some parts based on the comments, can you use the latest version of the files since some lines were already merged?

**Comment by Vilifisaur** [comment]
Ok I merged the corrections with latest version. Thanks for your help.

**Pull request closed without merging**